crates/diesel_models/src/payment_attempt.rs

```diff
@@ -43,22 +32,15 @@ pub struct PaymentAttempt {
     pub last_synced: Option<PrimitiveDateTime>,
     pub cancellation_reason: Option<String>,
     pub amount_to_capture: Option<MinorUnit>,
-    pub mandate_id: Option<String>,
     pub browser_info: Option<serde_json::Value>,
     pub error_code: Option<String>,
     pub payment_token: Option<String>,
     pub connector_metadata: Option<serde_json::Value>,
     pub payment_experience: Option<storage_enums::PaymentExperience>,
-    pub payment_method_type: Option<storage_enums::PaymentMethodType>,
     pub payment_method_data: Option<serde_json::Value>,
-    pub business_sub_label: Option<String>,
-    pub straight_through_algorithm: Option<serde_json::Value>,
     pub preprocessing_step_id: Option<String>,
-    // providing a location to store mandate details intermediately for transaction
-    pub mandate_details: Option<storage_enums::MandateDataType>,
     pub error_reason: Option<String>,
     pub multiple_capture_count: Option<i16>,
-    // reference to the payment at connector side
     pub connector_response_reference_id: Option<String>,
     pub amount_capturable: MinorUnit,
     pub updated_by: String,
```

Comment:
We have a field called encoded_data, what is the purpose of it? If that is required make it secret or encrypted.

Reply:
Add connector_id and avoid using merchant_connector_id